dist/image-sequencer.js

```diff
@@ -47644,11 +47644,15 @@ ImageSequencer = function ImageSequencer(options) {
     return this;
   }
 
-  function run(spinnerObj,t_image,t_from) {
-    let progressObj;
+  // Config is an object which contains the runtime configuration like progress bar
+  // information and index from which the sequencer should run
+  function run(config,t_image,t_from) {
+    let progressObj,index;
+
     if(arguments[0] != 'test'){
```

Comment:
Should this become `config.mode !== 'test'`?

Reply:
Yeah we can do that!!! wow Awesome idea!!!